src/components/events/EventForm.jsx

```diff
@@ -41,9 +43,13 @@ class EventForm extends React.Component {
 
   handleSubmit = async () => {
     const { eventId, formType } = this.props;
+    const { eventTitle, eventDate, numJudges } = this.state;
     const collectionName = 'events';
-    const data = _.omit(this.state, 'disabled');
-
+    const data = {
+      eventDate: new Date(eventDate),
```

Comment:
What is this behaviour if the eventDate is `''`? That's our default above, I'm not sure if this is safe behaviour for us to do. It appears to still be the same object as `_.omit(this.state, 'disabled');`.

I'm curious if maybe we could just keep this line as is and do the `new Date` somewhere else, maybe even just store the state as a `Date` object

Reply:
Let's do this for now because there is a default behaviour until we integrate in a date picker